torch/nn/modules/module.py

```diff
@@ -1248,6 +1335,8 @@ def __setstate__(self, state):
             self._non_persistent_buffers_set = set()
         if '_is_full_backward_hook' not in self.__dict__:
             self._is_full_backward_hook = None
+        if '_backward_pre_hooks' not in self.__dict__:
+            self._backward_pre_hooks = OrderedDict()
```

Comment:
Hmm why don't I see _backward_hooks here?

Reply:
I think `_backward_hooks` has existed long before (atleast since 1.0.1 based on test_conv_backcompat in test_nn) and hence old checkpoints will probably have that field.

Without the update, these tests fail from test_nn: test_module_backcompat, test_conv_backcompat

Reply:
Gotcha, thanks